src/domains/wallet/pages/WalletDetails/WalletDetails.Actions.test.tsx

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable @typescript-eslint/require-await */
+import { Contracts, ReadOnlyWallet } from "@ardenthq/sdk-profiles";
+import userEvent from "@testing-library/user-event";
+import { createHashHistory } from "history";
+import nock from "nock";
+import React from "react";
+import { Route } from "react-router-dom";
+
+import { WalletDetails } from "./WalletDetails";
+import { LedgerProvider } from "@/app/contexts";
+import { buildTranslations } from "@/app/i18n/helpers";
+import walletMock from "@/tests/fixtures/coins/ark/devnet/wallets/D8rr7B1d6TL6pf14LgMz4sKp1VBMs6YUYD.json";
+import {
+	env,
+	getDefaultProfileId,
+	MNEMONICS,
+	render,
+	RenderResult,
+	screen,
+	syncDelegates,
+	waitFor,
+} from "@/utils/testing-library";
+
+const translations = buildTranslations();
+
+const history = createHashHistory();
+let walletUrl: string;
+
+let profile: Contracts.IProfile;
+let wallet: Contracts.IReadWriteWallet;
+let unvotedWallet: Contracts.IReadWriteWallet;
+
+const renderPage = async ({ waitForTopSection = true } = {}) => {
+	const utils: RenderResult = render(
+		<Route path="/profiles/:profileId/wallets/:walletId">
+			<LedgerProvider>
```

Comment:
I think `LedgerProvider` is not needed here, as it is already included by with the `render` method. Same for the other files.

Reply:
@dated should be ok now. Thanks!